### PR TITLE
DEV: Move cursor to the end of fast-edit input

### PR DESCRIPTION
**Pull request by CvX**

### CODE REVIEW

File: app/assets/javascripts/discourse/app/components/fast-edit.hbs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 {{! template-lint-disable no-invalid-interactive }}
 <div class="fast-edit-container" {{on "keydown" this.onKeydown}}>
   <textarea
-    {{auto-focus}}
+    {{auto-focus cursorPosition="end"}}
```
Comment (jjaffeux) [inline app/assets/javascripts/discourse/app/components/fast-edit.hbs:5]: cool :+1:

**Comment by jjaffeux** [comment]
Writing a test for this probably wouldn't be too hard

**Review by jjaffeux — APPROVED** [review]

**Comment by ZogStriP** [comment]
@cvx can we get this merged?

**Comment by SamSaffron** [comment]
@CvX this is conflicting now

**Comment by SamSaffron** [comment]
I am closing this @CvX feel free to open another one.

**Pull request closed without merging**